azure_functions_worker/functions.py

```diff
@@ -41,6 +43,7 @@ def __init__(self, function_name: str, msg: str) -> None:
 
 class Registry:
     _functions: typing.MutableMapping[str, FunctionInfo]
+    _deferred_bindings_enabled: bool = False
```

Comment:
task to explore it to become a property in future

Reply:
Yes, added a task for it